portfolio/src/main/webapp/style.css

```diff
@@ -654,4 +554,75 @@ span {
 #short-comment {
     margin: auto;
     width: 190px;
-}
\ No newline at end of file
+}
+
+.margin-fix {
+    margin: auto;
+}
+
+@media screen and (min-width:765px) and (max-width:1900px)
```

Comment:
Are these like shared styles between the two? Little confused on the empty classnames here. Are you trying to clear the styles above?

Reply:
I forgot to remove this after moving the other file.